CHANGELIST.md

```diff
@@ -1,3 +1,12 @@
+## V 1.2.0
+Added spark 3.1 support. Added brackets to staging table name for NO_DUPLICATES mode.
+
+## v 1.0.0
```

Comment:
can we order it in lastest at the top. V1.0.0 should be at the bottom

Reply:
@shivsood It should be 1.1.0. Now fixed.